src/scripts/parsePrimaryDatePath.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I noticed that this script didn't have a "Wrote to..." logger like the other scripts did. Do we want to add that?